### ISSUE TITLE
Forum UI Improvements

### ISSUE THREAD

**Issue by arncv** [arncv/Intima#19]
Propose a more organized way of categorizing and sorting discussions in the forum.

**Comment by HamnaJabbar** [comment]
Hi @arncv ! will you plz let me know that by categorizing discussions in forum what do you mean?

**Comment by arncv** [comment]
Hey @HamnaJabbar! So, when we talk about categorizing discussions in the forum, we basically mean organizing all the user queries, conversations, and discussions into different 'topics' or 'categories'. This makes it super easy for users to navigate through the forum.

Here's a little example to explain it better. Let's say our forum has a bunch of discussions on topics like 'Sexual Orientation,' 'Birth Control,' and 'Sexual Violence'. Instead of piling them all up in one place, we can create separate forums for each topic. So, if someone wants to get into the discussions about 'Sexual Orientation,' they can simply go straight to that specific category.

In a nutshell, it's like giving the forum page a fresh new look and making it way more user-friendly.

**Comment by HamnaJabbar** [comment]
Got it, will you plz assign this to me? 
Thankyou

**Comment by HamnaJabbar** [comment]
Hi @arncv i have updated the ForumFAQUI.
